src/components/Game.js

```diff
@@ -15,108 +16,194 @@ class Game extends Component {
-      console.log(`Random new noteIndex in while loop: ${newNoteIndex}`);
     }
+
     this.setState({lastPromptedNote: lastNote, promptedNote: newNote});
-    console.log(`new noteNum: ${newNote}, lastNoteNum: ${lastNote}`);
-    let noteObject = newNote;
 
 
-    const difficultyModifier = 0.5
-    const targetFreq = noteObject.frequency
+    const difficultyModifier = 0.7
+    const targetFreq = newNote.frequency
     let targetFreqRangeLower = (targetFreq-(targetFreq*0.02806)*difficultyModifier)
     let targetFreqRangeUpper = (targetFreq+(targetFreq*0.02973)*difficultyModifier)
 
     this.setState({
-      // promptedNoteLetter: noteObject.noteName,
-      promptedNoteLetter: noteObject.noteNameOctave, // CHANGE THIS ONCE GRAPHPICS ARE IN!
-      promptedNoteFreq: noteObject.frequency,
+      promptedNoteLetter: newNote.noteName,
+      // promptedNoteLetter: newNote.noteNameOctave, // @@@@@ DELETE THIS ONCE GRAPHPICS ARE IN! @@@@@
+      promptedNoteFreq: newNote.frequency,
       targetFreqRangeLower: targetFreqRangeLower,
       targetFreqRangeUpper: targetFreqRangeUpper
-    }
-  );
-  }
+    });
+
+    this.props.increaseNotesTriedCallback();
+
+    devLogger(`*** NEW NOTE *** `);
+    if (lastNote) {devLogger(`lastNote: ${lastNote.noteNum}`);}
+    if (newNote) {devLogger(`newNote: ${newNote.noteNum}`);}
+  };
 
   getRandomIntInclusive(min, max) {
       const randInt = Math.floor(Math.random() * (max - min + 1)) + min;
-      console.log('randInt in getRandomIntInclusive: ', randInt);
       return randInt;
   }
 
+
+
+
+  // *** GET USER PITCH ***
   getuserPlayingPitch = (pitch) => {
     const oldPitch = this.state.userPlayingPitch;
+    devLogger(`oldPitch: ${oldPitch}, newPitch: ${pitch}`);
+    this.checkPitchChange(oldPitch, pitch);
+  };
+
+
+  // *** GAME LOGIC ***
+  checkPitchChange(oldPitch, pitch) {
+    if (oldPitch !== pitch ){ // pitch has changed, A or B
+      this.setState({userPlayingPitch: pitch}, ()=>{devLogger(`pitch changed to: ${pitch}`)});
+      this.handlePitchChange(pitch);
+    } else { // pitch has not changed, C
+      this.handlePitchNoChange(pitch);
+    };
+  }
+
+  checkPitchInRange = (pitch) => {
+    if (pitch <= this.state.targetFreqRangeUpper && pitch >= this.state.targetFreqRangeLower) {return true} else {return false};
+  };
+
+  checkTimerStarted = () => {
+    if (this.state.targetTime) {return true} else {return false};
+  }
 
-    if (oldPitch !== pitch ){
-      this.setState({userPlayingPitch: pitch}, ()=>{
-        // console.log('in game, userPlayingPitch: ', this.state.userPlayingPitch);
-        if (pitch <= this.state.targetFreqRangeUpper && pitch >= this.state.targetFreqRangeLower) {
-          this.toggleNoteMatched();
+  handlePitchChange = (pitch) => {
+    let pitchInRange = this.checkPitchInRange(pitch); // true or false
+    let timerStarted = this.checkTimerStarted();
+    let targetTimeReached = this.checkTargetTimeReached();
+
+    if (pitchInRange) { // A
```

Comment:
this logic is a bit involved, can we simplify it? its hard to read hence hard to maintain.

Reply:
Let's put this on our list of things to discuss on Friday. The whole decision tree is pretty complex.

Reply:
Sure, hoever keen in mind that complex logic should not mean complex code ;)